index.html

```diff
@@ -376,8 +384,9 @@
 			// Usually, we add everyone to a blacklist then remove people who meet the criteria. 
 
 			// Work URLS
+			
 			withoutWorkURL.push(person);
-			if (typeof person['contact'] !== 'undefined' && person['contact'].length > 0) {
+			if (checkNotUndefinedandLength(person, 'contact') > 0) {
```

Comment:
If checkNotUndefinedandLength returns a True or False, you won't need the `> 0` in the `if`.